pkg/store/prometheus.go

```diff
@@ -220,6 +225,55 @@ func (p *PrometheusStore) Series(r *storepb.SeriesRequest, s storepb.Store_Serie
 	return p.handleStreamedPrometheusResponse(s, httpResp, queryPrometheusSpan, extLset)
 }
 
+func (p *PrometheusStore) queryPrometheus(s storepb.Store_SeriesServer, r *storepb.SeriesRequest) error {
+	ctx := context.Background()
+	queryOpts := promclient.QueryOptions{
```

Comment:
I think in general these options don't do anything because here we are querying Prometheus and these options are only understandable by Thanos Query :smile: